framework/validators/ExistValidator.php

```diff
@@ -137,14 +139,41 @@ private function prepareConditions($targetAttribute, $model, $attribute)
             if ($this->allowArray) {
                 throw new InvalidConfigException('The "targetAttribute" property must be configured as a string.');
             }
-            $params = [];
+            $conditions = [];
             foreach ($targetAttribute as $k => $v) {
-                $params[$v] = is_int($k) ? $model->$v : $model->$k;
+                $conditions[$v] = is_int($k) ? $model->$v : $model->$k;
             }
         } else {
-            $params = [$targetAttribute => $model->$attribute];
+            $conditions = [$targetAttribute => $model->$attribute];
         }
-        return $params;
+
+        if (!$model instanceof ActiveRecord) {
+            return $conditions;
+        }
+
+        // Add table prefix for column
```

Comment:
this comment looks to be bound solely for the next line and is confusing. A private method for the whole next block would improve readability